teek/_widgets/treeview.py

```diff
@@ -0,0 +1,452 @@
+import functools
+from collections.abc import MutableSequence, Iterable
+
+import teek
+from teek._widgets.base import Widget, ChildMixin
+from teek._structures import ConfigDict
+from teek._tcl_calls import make_thread_safe
+
+
+class FallbackConfigDict(ConfigDict):
+    """
+    Config dictionary which uses fallback values if the given handler
+    function cannot be executed
+    """
+
+    def __init__(self):
+        super().__init__()
+
+        # Handler functions
+        self._handlers = {}
+
+        # Fallback values (used if handler function cannot be executed)
+        self._fallbacks = {}
+
+    def _set(self, option, value):
+        try:
+            # Set value of fallback
+            self._fallbacks[option] = value
+            # Look for handler function(s)
+            handlers = (self._handlers[option] if option in self._handlers
+                        else self._handlers['*'])
+
+            if not isinstance(handlers, Iterable):
+                handlers = [handlers]
+
+            for handler in handlers:
+                handler(None, option, value)
+        except Exception:
+            pass
+
+    def _get(self, option):
+        try:
+            # Look for handler function
+            handler = (self._handlers[option]
+                       if option in self._handlers else self._handlers['*'])
+
+            if isinstance(handler, Iterable):
+                handler = handler[0]
+
+            return handler(self._types.get(option, str), option)
+        except Exception:
```

Comment:
Same as above, which kinds of Exceptions are we guarding against here?

Reply:
Same answer as above, but problem is solved by change mentioned in the comment below.